content/en/boilerplates/notes/1.3.md

```diff
@@ -0,0 +1,79 @@
+- **Improved** load balancing to direct traffic to the [same region and zone](/faq/traffic-management/#controlling-inbound-ports) by default.
+- **Improved** Pilot by reducing CPU utilization. The reduction approaches 90% depending on the specific deployment.
+- **Improved** the `ServiceEntry` API to allow for the same hostname in different namespaces.
+- **Improved** the [Sidecar API](/docs/reference/config/networking/v1alpha3/sidecar/#OutboundTrafficPolicy) to customize the `OutboundTrafficPolicy` policy.
+
+## Security
+
+- **Added** a unified logging pattern to Citadel agent.
+- **Added** trust domain validation for Envoy proxies using mutual TLS. Improved validation only applies to the `Sidecar` resource.
+- **Added** metrics to the `/metrics` endpoint of Citadel Agent on port 15014 to monitor the SDS service.
+- **Added** diagnostics to Citadel agent using the `ctrlZ` endpoint on port 9876.
+- **Added** diagnostics to Citadel agent using the `/debug/sds/workload` and `/debug/sds/gateway` on port 8080.
+- **Added** metrics to the `/metrics` endpoint of Citadel Agent on port 15014 to monitor the SDS service.
+- **Added** trust domain validation for Envoy proxies using mutual TLS. Improved validation only applies to the `Sidecar` resource.
+- **Added** [labels](/docs/concepts/security/#how-citadel-determines-whether-to-create-service-account-secrets) to control service account secret generation by namespace.
+- **Added** SDS support to deliver key and certificates to the Istio control plane.
+- **Improved** the ingress gateway to [load the trusted CA certificate from a separate secret](/docs/tasks/traffic-management/ingress/secure-ingress-sds/#configure-a-mutual-tls-ingress-gateway) when using SDS.
+- **Improved** SDS security by enforcing the usage of [Kubernetes Trustworthy JWTs](/blog/2019/trustworthy-jwt-sds).
+- **Removed** support for Istio SDS when using [Kubernetes versions earlier than 1.13](/blog/2019/trustworthy-jwt-sds).
+- **Removed** integration with Vault CA temporarily. SDS requirements caused the temporary removal but we will reintroduce Vault CA integration a future release.
+- **Enabled** the Envoy JWT filter by default to improve security and reliability.
+
+## Telemetry
+
+- **Added** Access Log Service [ALS](https://www.envoyproxy.io/docs/envoy/latest/api-v2/service/accesslog/v2/als.proto#grpc-access-log-service-als) support for Envoy gRPC.
+- **Added** a Grafana dashboard for Citadel monitoring.
+- **Added** [metrics](/docs/reference/commands/sidecar-injector/#metrics) for monitoring the sidecar injector webhook.
+- **Added** control plane metrics to monitor Istio's configuration state.
+- **Added** telemetry reporting for traffic destined to the `Passthrough` and `BlackHole` clusters.
+- **Added** alpha support for in-proxy generation of service metrics using Prometheus.
```

Comment:
link to docs ?

Reply:
@mandarjog

Reply:
`alpha support for in-proxy generation of *http* service metrics using Prometheus.` 
https://github.com/istio/istio/wiki/Mixerless-HTTP-Telemetry

Reply:
just to be clear, I am not linking to a wiki in our release notes.